public/index.html

```diff
@@ -34,8 +34,9 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="header"></div>
-    <div id="stat_blocks"><div>
+    <div id="root"></div>
+
+    <div id='creature-compendium'></div>
```

Comment:
ACS-01: This div is unneeded since the app component adds the compendium component